.prettierrc.yaml

```diff
@@ -1,2 +1,5 @@
-printWidth: 80
+semi: false
```

Comment:
Should we keep using no semicolons or should we switch to Prettier defaults and start using semicolons again?

https://prettier.io/docs/options#semicolons

I slightly prefer to use semicolons for consistency with Rust, where a statement with a semicolon has a different meaning than a statement without one.

I believe the default TypeScript formatter uses semicolons too. But they also use double quotes, IIRC, while we are using single quotes, so this may not be the best example.

Reply:
My preference is without semicolons

Reply:
I would vote in favor of no semicolons. Personally I consider code with no semicolons more readable.